shared/oae/errors/js/accessdenied.js

```diff
@@ -30,7 +30,7 @@ require(['jquery','oae.core'], function($, oae) {
         }
 
         // After signing in, the user should be redirected to the redirect target encoded in the URL
-        var redirectUrl = $.url().param('url') || '/me';
+        var redirectUrl = oae.api.util.url().param('url') || '/me';
 
```

Comment:
When the user is not logged in, the access denied page will show a Sign In button that is currently not working. When access is denied and the user is not logged in, I think it makes sense to trigger sign in automatically (either launch the modal or forward to the external authentication strategy).